docs/topics/collection-transformations.md

```diff
@@ -144,7 +144,7 @@ fun main() {
 {kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
 
 For building maps with collection elements as values, there is the function [`associateBy()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/associate-by.html).
-It takes a function that returns a key based on an element's value. If two elements are equal, only the last one remains
+It takes a function that returns a key based on an element's value. If two elements' keys are equal, only the last one remains
```

Comment:
Good point, thanks!

Reply:
Perhaps we can show this case in the example as well. Just an idea, I don't insist.

Reply:
Good idea, but maybe later) Could you approve this PR, please?